src/graph/parser.ts

```diff
@@ -135,6 +137,25 @@ export class Graph {
         }
     }
 
+    private static parseSize(cssString: string):{value: string, isDynamic: boolean} {
+        cssString = cssString.toString()
```

Comment:
This doesn't seem to do anything

Reply:
oh that was because when i ran it before, somehow numbers were getting passed into the function on accident? i still have no clue why but it was fixed by converting it to a string. Maybe some other changes fixed it but thats how i patched that problem earlier